src/openrct2/world/footpath.c

```diff
@@ -587,9 +587,9 @@ void game_command_remove_footpath(int *eax, int *ebx, int *ecx, int *edx, int *e
 	*ebx = footpath_remove_real((*eax & 0xFFFF), (*ecx & 0xFFFF), (*edx & 0xFF), (*ebx & 0xFF));
 }
 
-money32 footpath_place(int type, int x, int y, int z, int slope, int flags)
+money32 footpath_place(int type, int x, int y, int z, int slope, int flags, uint16 pathScenery)
 {
-	return game_do_command(x, (slope << 8) | flags, y, (type << 8) | z, GAME_COMMAND_PLACE_PATH, 0, 0);
+	return game_do_command(x, (slope << 8) | flags, y, (type << 8) | z, GAME_COMMAND_PLACE_PATH, pathScenery, 0);
```

Comment:
GAME_COMMAND_PLACE_PATH is used for placing a path and for placing path scenery. In place_path(...) this was just set to zero, as placing path scenery is usually done with the scenery tools.